Demo/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <!-- Trust user added CAs while debuggable only -->
```

Comment:
Do we still need this?

Reply:
Yep I believe this is still needed. @saperi22 was having cert issues on his emulator and needed to have the demo app trust the user added CA.